main_pullrequest.go

```diff
@@ -24,7 +24,7 @@ func processGitHubPullRequest(ctx *gin.Context, pr *github.PullRequestEvent, git
 	req := pr.GetPullRequest()
 	base := req.GetBase()
 	head := req.GetHead()
-	prRef = base.GetRef()
+	prRef = head.GetRef()
 
```

Comment:
@alfrunes Actually, after my second commit, `prRef` does not need to be defined here anymore. Should I clean it up?